2019/MDAnalysis_ideas_list.md

```diff
@@ -1 +1,84 @@
+     2. **Working with Groups** (requires _Working with AtomGroups_): The "container" hierarchy (`Universe` > `Segment` > `Residue` > `Atom`) and the corresponding groups `SegmentGroup`, `ResidueGroup`, `AtomGroup`: commonalities and differences, aggregating methods. How to work with `fragments`.
+     3. **Writing selections**: outputting selections for other codes
+     4. **Working with topology information**: introduction to the topology system; how to work with bonds; identify bonded atoms; working with angles and dihedrals; selections by type
+  3. Advanced level
+     1. **System building** (requires _Working with topology information_): how to add atoms or bonds or create simple topologies from scratch; generating initial coordinates
+     2. **Extending file reading with own code** (requires _System building_): write a Reader for once own custom file format and dynamically add it to MDAnalysis
+     3. **Write your own analysis class**: shows how to leverage the [MDAnalysis.analysis.AnalysisBase](https://www.mdanalysis.org/docs/documentation_pages/analysis/base.html) class to create feature-full custom analysis tools.
+
+For this and other documents we want to start adding example Jupyter notebooks (such as the first few [example notebooks](https://github.com/MDAnalysis/binder-notebook/tree/master/notebooks)) to our sphinx-based restructured text documentation via the [nbsphinx](https://nbsphinx.readthedocs.io/en/0.4.2/) extension.
+
+We also want to **include more diagrams, pictures, and graphs** to make clearer what the relationships between different parts of the code are and what output might look like.
+
+### Howto Library
+
+We wand to create a library of short *Howtos* (mini-tutorials) that quickly show to obtain a specific result. The ideal format are Jupyter Notebooks with runable code. We already have a few [example notebooks](https://github.com/MDAnalysis/binder-notebook/tree/master/notebooks) that can [run on Binder](https://mybinder.org/v2/gh/MDAnalysis/binder-notebook/master?filepath=notebooks) but we are not covering a sufficient range of examples and examples might be outdated. Ideally we would regularly test if the notebooks still work.
+
+1. Developers need to identify a small number of "FAQ" cases that would make for good small self-contained Howtos
+2. Initial selection of examples
+     - B-factor coloring
+     - PDB file manipulation (e.g. adding ChainIDs)
+     - selections with updating AtomGroups
+     - trajectory conversion
+     - RMSD analysis
+     - RMSF analysis
+     - dihedral analysis
+3. Jupyter notebooks need to be written that solve the problem. These notebooks should use example data and should be executable on binder. They can use either data files from MDAnalysisTests or MDAnalysisData.
+4. For each case, a short HowTo document (in restructured text and processed with sphinx) needs to be written that 
```

Comment:
contradicts the requirement to use jupyter notebooks

Reply:
I wasn't clear - I was thinking about short narrative that then includes the notebook.

However, I will remove project 3 from the proposal and focus on 1 and 2.